pkg/services/rpcsrv/server.go

```diff
@@ -2185,7 +2184,16 @@ func (s *Server) prepareInvocationContext(t trigger.Type, script []byte, contrac
 
 		err = s.chain.InitVerificationContext(ic, contractScriptHash, &transaction.Witness{InvocationScript: script, VerificationScript: []byte{}})
 		if err != nil {
-			return nil, neorpc.NewInternalServerError(fmt.Sprintf("can't prepare verification VM: %s", err))
+			switch {
+			case errors.Is(err, core.ErrUnknownVerificationContract):
+				return nil, neorpc.WrapErrorWithData(neorpc.ErrUnknownContract, err.Error())
+			case errors.Is(err, core.ErrInvalidVerificationContract):
+				return nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidVerificationFunction, fmt.Sprintf("Missing verify method: %s", err.Error()))
+			case errors.Is(err, core.ErrInvalidVerification):
+				return nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidVerificationFunction, fmt.Sprintf("Invalid verify return value: %s", err.Error()))
```

Comment:
`core.ErrInvalidVerification` is returned on invalid witness verification script (and it's not related to the verification contract return value). And here we have an empty verification script (take a look at L2185). Thus, this is a dead code, remove this case, please.